enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Visualization.tsx

```diff
@@ -33,19 +37,23 @@ export const QuestionVisualization = () => {
   const card = question.card();
 
   return (
-    <QueryVisualization
-      className={cx(CS.flexFull, CS.fullWidth, CS.fullHeight)}
-      question={question}
-      rawSeries={[{ card, data: result && result.data }]}
-      isRunning={isQueryRunning}
-      isObjectDetail={false}
-      isResultDirty={false}
-      isNativeEditorOpen={false}
-      result={result}
-      noHeader
-      mode={mode}
-      navigateToNewCardInsideQB={navigateToNewCard}
-      onNavigateBack={onNavigateBack}
-    />
+    <Box w="100%" h="100%" ref={ref}>
+      <Box w={width} h={height || (width ? width / 9 : 500)}>
```

Comment:
I wonder if we should make the height calculation into a function, we might have similar duplicated logic that handles the visualization height.

Reply:
ah that's true. I'll do that